src/pages/chart-view/chart-view.js

```diff
@@ -62,6 +68,16 @@ const ChartView = (props) => {
     )
   }
 
+  const onUpdate = (data, layout, frames) => {
+    var dataClone = cloneDeep(data)
+    var dataSourceClone = cloneDeep(dataSources)
+
+    updateState(data, layout, frames)
```

Comment:
Why do we still keep a local copy of all of this plotly state if we're sending it all to redux?

Reply:
Yep good point